pkg/controllers/provisioning/scheduling/topology.go

```diff
@@ -70,9 +71,11 @@ func NewTopology(
 	nodePools []*v1.NodePool,
 	instanceTypes map[string][]*cloudprovider.InstanceType,
 	pods []*corev1.Pod,
+	opts ...Options,
```

Comment:
Do we need all options passed?

Reply:
I'd like to keep them just because it's a little more sensible than a boolean and it makes it have nice parity with the scheduler